Controllers/BasicController.php

```diff
@@ -1,7 +1,7 @@
 <?php
-
 namespace Controllers;
 
+
```

Comment:
Please remove any changes to this file. It should not have visible changes for this task.